.github/workflows/ci.yml

```diff
@@ -31,11 +31,15 @@ jobs:
   detect-changes:
     outputs:
       actionlint: ${{ steps.filter.outputs.actionlint }}
```

Comment:
CodeQL check failing due to not explicitly mentioning workflow permissions: https://github.com/camunda/camunda/pull/28035/checks?check_run_id=37095150538

Reply:
I looked at it and consider it a false positive so I closed https://github.com/camunda/camunda/security/code-scanning/5320 with the following comment:

> This job [meant is the tasklist-frontend-tests job in Unified CI] does nothing besides calling a reusable workflow, and all jobs in that workflow use least-privilege for the token as verified by our custom rules.